CMakeLists.txt

```diff
@@ -122,21 +122,29 @@ endif()
 
 ## The library itself
 
+### DDC legacy target
 add_library(DDC INTERFACE)
-target_compile_features(DDC INTERFACE cxx_std_17)
+add_library(DDC::DDC ALIAS DDC)
+install(TARGETS DDC EXPORT DDCTargets)
+
+add_library(ddc INTERFACE)
+add_library(DDC::ddc ALIAS ddc)
+install(TARGETS ddc EXPORT DDCTargets)
+target_compile_definitions(ddc INTERFACE MDSPAN_USE_PAREN_OPERATOR=1)
+if("${DDC_BUILD_DOUBLE_PRECISION}")
+    target_compile_definitions(ddc INTERFACE DDC_BUILD_DOUBLE_PRECISION)
```

Comment:
I usually prefer the generation of a config.h that is installed. From my point of view, it leaves less room for errors on the user side, especially if not using cmake.

Reply:
I agree, see #672. Do we want to support another configuration tool ?

Reply:
Not support it, but there's no good reason to do otherwise and it has mostly only advantages.